.circleci/config.yml

```diff
@@ -265,6 +304,25 @@ workflows:
             branches:
               only: /^master$/
 
+  release_uat:
+    jobs:
```

Comment:
```suggestion
    jobs:
      filters:
        branches:
          ignore:
            - master
            - gh-pages
        tags:
          ignore: /.*/
```

Reply:
this is to ignore commits on the master branch and gh-pages branch as well as each and every tag.
Basically this job will work on PR level and after approval as you implemented.

Reply:
Ref: https://support.circleci.com/hc/en-us/articles/115015953868-Filter-workflows-by-branch-